obspy/core/tests/test_stats.py

```diff
@@ -281,7 +282,7 @@ def test_casted_stats_nscl_writes_to_mseed(self):
         stats_items = set(Stats())
         new_stats = Stats()
         new_stats.__dict__.update({x: st[0].stats[x] for x in stats_items})
-        with warnings.catch_warnings(record=True):
+        with pytest.warns(UserWarning):
```

Comment:
Any match matching here needed to ensure the correct warnings are being raised?

Reply:
Could be done, but not really necessary, because only a small amount of code is tested. The goal of the code before was to simply ignore the warnings (not test for), but it was not sufficient, because filterwarnings('ignore') was missing.